.ls-lint.yml

```
@@ -0,0 +1,21 @@
+  server/handlers/*/analyzer:
+    .go: snake_case
+
+  server/handlers/*/lsp:
+    .go: kebab-case
+
+  # Generated by antlr
+  server/handlers/*/ast/parser:
+    .go: snake_case
+
+ignore:
+  - .git 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)